packages/web-components/fast-foundation/src/radio-group/radio-group.template.ts

```diff
@@ -17,11 +16,7 @@ export function radioGroupTemplate<T extends FASTRadioGroup>(): ElementViewTempl
             @focusout="${(x, c) => x.focusOutHandler(c.event as FocusEvent)}"
         >
             <slot name="label"></slot>
-            <div
-                class="positioning-region ${x =>
-                    x.orientation === Orientation.horizontal ? "horizontal" : "vertical"}"
-                part="positioning-region"
-            >
+            <div class="control" part="control">
```

Comment:
We still have other examples of positioning region, what makes this a control vs. the example in slider?

Reply:
I didn't try to remove it in slider because I think the nesting and positioning are important to get the functionality, relying on the margins on the inner element. In general, my approach was to not have a "positioning" or any specifically named container elements when possible. Per the rules from our meeting, we landed on "control" being the word for the main portion that is the same as the actual component (aside from the `template` being the thing).

Said another way, the goal is to have only a single inner container if necessary, which is named "control" because it's not the `template` of the component itself. We decided that duplicating the name was more confusing, like a "tooltip" component with a "tooltip" div. I'd love to get rid of it in slider as well, but I think it won't be easy, so that was an _exception_.